lib/puppet/provider/package/pip.rb

```diff
@@ -40,11 +40,7 @@ def self.instances
   end
 
   def self.cmd
-    if Facter.value(:osfamily) == "RedHat" and Facter.value(:operatingsystemmajrelease).to_i < 7
-      "pip-python"
-    else
-      "pip"
-    end
+    "pip"
```

Comment:
Well actually the issue is that the latest python-pip package in EPEL repo and the only available one doesn't have /usr/bin/pip-python symbolic link anymore.

However you are right, I've check some old python-pip packages
one of them has only /usr/bin/pip-python
and another one has both /usr/bin/pip and /usr/bin/pip-python
and the latest one has only /usr/bin/pip

Anyway I've addressed this issue and added a backward compatibility for pip-python binary aswell